test/zdtm/static/bpf_array.c

```diff
@@ -1,5 +1,6 @@
 #include <linux/bpf.h>
 #include <bpf/bpf.h>
+#include <bpf/libbpf_common.h>
```

Comment:
It seems this is redundant -- `bpf.h` explicitly includes `libbpf_common.h` (checked with v0.4 and v0.7)